lib/rubocop/cop/rspec/redundant_context.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Detect redundant `context` hook.
+      #
+      # @example
+      #   # bad
+      #   context 'when condition' do
+      #     it 'tests something' do
+      #     end
+      #   end
+      #
+      #   # good
+      #   it 'tests something when condition' do
+      #   end
+      #
+      class RedundantContext < Base
+        MSG = 'Redundant context with single example.'
+
+        # @!method redundant_context?(node)
+        def_node_matcher :redundant_context?, <<~PATTERN
+          (block
+            (send #rspec? :context _)
```

Comment:
Just specifically `context` and `it`?
I’dd love to have at least balancing examples for describe/example aliases if it’s a deliberate design choice.